dist_tables/dml.rst

```diff
@@ -8,23 +8,25 @@ The following code snippets use the Github events example, see :ref:`ddl`.
 Inserting Data
 --------------
 
-Single row inserts
-$$$$$$$$$$$$$$$$$$
+Inserting Row Literals
```

Comment:
Echoing Ozgun's comment, we can probably leave our basic INSERT example as-is. We could add a separate section (multi-row inserts), and expand a bit on why you may want to do this along with examples.